### PR TITLE
setup: copy osrd eslintrc rules

### LINKED ISSUE #279: setup: copy osrd eslintrc rules to osrd-ui
- [x] copy eslintrc rules from osrd to osrd-ui
- [x] add global script for sub-modules eslint scripts
- [x] add lint:fix script for sub-modules
- [x] fix all auto-fixable warnings/errors

### PR DESCRIPTION
**Pull request by Yohh**
- update eslintrc
- add global lint script in /package.json
- add lint & lint:fix script in /<sub-module>/p	package.json
- fix auto-fixables warnings/errors

close #279

### CODE REVIEW

**Review by sim51 — APPROVED** [review]

**Comment by sim51** [comment]
All good for me. 
Great job !

File: .eslintrc [hunk 1]
```diff
@@ -1,31 +1,151 @@
...
   "settings": {
     "react": {
-      "version": "detect",
-    },
-  },
-}
+      "version": "detect"
+    }
+  }
+}
```
Comment (kmer2016) [inline .eslintrc]: EOF ;)
```suggestion
}

```

**Review by kmer2016 — APPROVED** [review]
LGTM, good job !

File: .eslintrc (hunk 1 above)
Comment (Yohh) [reply .eslintrc]: good catch! done

**Pull request merged**